packages/interactivity-router/src/assets/styles.ts

```diff
@@ -259,12 +259,14 @@ export const applyStyles = ( styles: StyleElement[] ) => {
 	window.document
 		.querySelectorAll( 'style,link[rel=stylesheet]' )
 		.forEach( ( el: HTMLLinkElement | HTMLStyleElement ) => {
-			if ( styles.includes( el ) ) {
-				const { originalMedia = 'all' } = el.dataset;
-				el.sheet.media.mediaText = originalMedia;
-				el.sheet.disabled = false;
-			} else {
-				el.sheet.disabled = true;
+			if ( el.sheet ) {
```

Comment:
Could it be that, at this specific moment, there is a `link` element that is still loading and whose `sheet` property does not yet exist, but is created right after?

Reply:
It could happen for a `link` element that shouldn't be active on the page you are navigating to, which is not a problem because it should have media `preload` and won't affect the styles.

The `link` elements that belong to the new page should be ready at that point.

Reply:
Good enough. Thanks 🙂